text/2019-02-26-tikv-client-command-line.md

```diff
@@ -0,0 +1,232 @@
+# TiKV Rust Client Command Line Tool
+
+## Summary
+
+Use the TiKV Rust Client to build a simple command line tool for inspection and anaylsis.

```

Comment:
```suggestion
This article describes how to the TiKV Rust Client to build a simple command line tool for inspection and analysis.
```

Reply:
This is an RFC, so this is a proposed action, not an instruction guide for someone. :) Thanks for the analysis correction!